lib/instance-validator.js

```diff
@@ -271,8 +271,7 @@ InstanceValidator.prototype._invokeBuiltinValidator = Promise.method(function(va
   }
 
   if (!validator[validatorType].apply(validator, [value].concat(validatorArgs))) {
-  // extract the error msg
-    throw test.msg || 'Validation ' + validatorType + ' failed';
+    throw new Error(test.msg || 'Validation ' + validatorType + ' failed');
```

Comment:
Yes please, we do not give you meaningful debug information if you throw non-errors.